frontend/src/pages/Landing/Payments/LandingPaymentsPage.tsx

```diff
@@ -1,65 +1,189 @@
+
+  const [onboardingHelperText, setOnboardingHelperText] = useState<
+    onboardingHelperTextType | undefined
+  >()
+
+  const mainSectionTextColour = 'white'
+
+  const handleSubmit = useCallback(async () => {
+    const fieldValue = formMethods.getValues(fieldId) as VerifiableFieldValues
+    try {
+      await sendOnboardingEmailMutation.mutateAsync({
+        email: fieldValue.value,
+      })
+      formMethods.clearErrors(fieldId)
+      setOnboardingHelperText(onboardingSuccessHelperText)
+    } catch (error) {
+      if (error instanceof Error) {
+        if (error.message === 'Forbidden') {
+          formMethods.setError(fieldId, {
+            message:
+              'We are unable to send you our guide because your email is not whitelisted on FormSG.',
+          })
+        } else {
+          formMethods.setError(fieldId, {
+            message: 'Something went wrong. Please try again later.',
+          })
+        }
+      }
+    }
+  }, [sendOnboardingEmailMutation, formMethods])
+
+  const bottomCTARef = useRef<HTMLDivElement | null>(null)
+
+  const LinkToBottomCTA = () => (
+    <Link
+      onClick={() =>
+        bottomCTARef.current?.scrollIntoView({ behavior: 'smooth' })
+      }
+    >
+      Receive our guide for more information.
+    </Link>
+  )
 
   return (
     <>
       <AppPublicHeader bg="primary.500" />
       <LandingSection bg="primary.500" pt={{ base: '2rem', md: 0 }} px="0">
         <Stack
           direction={{ base: 'column', lg: 'row' }}
-          align="center"
           spacing={{ base: '1.5rem', md: '3.125rem', lg: '7.5rem' }}
           pl={{ base: '1.5rem', md: '5.5rem', lg: '9.25rem' }}
         >
           <Flex
             flexDir="column"
             flex={1}
             pr={{ base: '1.5rem', md: '5.5rem', lg: '0' }}
+            mt={isMobile ? undefined : '2rem'}
           >
             <Text
               as="h1"
               textStyle={{ base: 'display-1-mobile', md: 'display-1' }}
-              color="white"
+              color={mainSectionTextColour}
             >
               Collect payments on your form
             </Text>
-            <SectionBodyText color="white">
+            <SectionBodyText color={mainSectionTextColour}>
               Citizens can now pay for fees and services directly on your form.
+              Enter your agency email to receive our guide on how to get started
+              with payments.
             </SectionBodyText>
-            <Box mt="2.5rem">
-              <Button
-                variant="reverse"
-                isFullWidth={isMobile}
-                as="a"
-                href={GUIDE_PAYMENTS_PUBLIC}
-                target="_blank"
-              >
-                Learn more about payments
-              </Button>
-            </Box>
+            <FormProvider {...formMethods}>
+              <Flex alignItems="start" mt="2rem">
+                <EmailField
```

Comment:
This EmailField contains the same prop as the one below right? Maybe we can just abstract the props up above and ... it? Or just wrap it around another component (in this .tsx) like `LandingPaymentEmailField`?

Reply:
that's sad. idt it's worth debugging if it's taking too much time, maybe can leave a comment

Reply:
Omg wth 😅, agree with wanling maybe we just leave a comment (so that any future eng that changes the landing page will know to change both `EmailField`)

Reply:
Correction: it blurs whenever there's a change in error state HAHA but yea the problem's still there

Reply:
Added comment for this in [d16a604](https://github.com/opengovsg/FormSG/pull/6462/commits/d16a60426584394a2e78a4ace02365967299d2f9)!